scripts/auth.js

```diff
@@ -0,0 +1,114 @@
+function errorMessage(target, message) {
+  const parent = target.parentElement;
+  const exists = parent.querySelector(".errorMessage");
+  if (!exists) {
+    const error = document.createElement("div");
+    error.classList.add("errorMessage");
+    error.textContent = message;
+    target.classList.add("wrongCssBorder");
+    target.after(error);
+  }
+}
+
+function clearError(input) {
+  const parent = input.parentElement;
+  const error = parent.querySelector(".errorMessage");
+  if (error) error.remove();
+  input.classList.remove("wrongCssBorder");
+}
+const loginInputs = document.querySelectorAll(".email_password_input");
+const loginButton = document.querySelector(".login_button");
+
+function loginCheck(input) {
+  clearError(input);
+
+  if (!input.value) {
+    if (input.placeholder.includes("이메일")) {
+      errorMessage(input, "이메일을 입력해주세요");
+    } else {
+      errorMessage(input, "비밀번호를 입력해주세요");
+    }
+  }
+}
+
+loginInputs.forEach((input) => {
+  input.addEventListener("focusout", (e) => {
+    loginCheck(e.target);
+  });
+});
+
+if (loginButton) {
+  loginButton.addEventListener("click", (e) => {
+    e.preventDefault();
+    let valid = true;
+
+    loginInputs.forEach((input) => {
+      loginCheck(input);
+      if (!input.value) valid = false;
+    });
+  });
+}
```

Comment:
💬 여담
loginButton이 있을 경우 아래처럼 옵셔널 체이닝을 이용해서 코드를 작성하실 수 있습니다~

```suggestion
loginButton?.addEventListener("click", (e) => {
  e.preventDefault();
  let valid = true;
  
  loginInputs.forEach((input) => {
    loginCheck(input);
    if (!input.value) valid = false;
  });
});
```

[mdn](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Optional_chaining)